CONTRIBUTE.md

```diff
@@ -0,0 +1,49 @@
+# TIMRLINK CLI Datenbank Migrationen 
+​
+## timrlink.net.CLI.Migrations
+​
+Enthält das Datenmodell und den vollständigen Datenzugriff über Entity Framework Core.
+​
+### Vorbereitungen
+
+Zu Beginn müssen die dotnet ef tools installiert werden.
+
+https://learn.microsoft.com/en-gb/ef/core/cli/dotnet
+
+Das hab ich direkt in Rider als Paket (Nu Get) hinzugefügt (War einfacher).
+
+```
+dotnet add package Microsoft.EntityFrameworkCore.Design
+```
+
+### Vorbereitungen für Apple Silicon Macs
+
+Zu Beginn hatte ich Probleme dotnet ef auszuführen. Nach einiger Recherge bin ich darauf gestossen:
+
+https://stackoverflow.com/questions/70929949/on-mac-m1-machine-not-able-to-run-ef-core-migrations-add-update-in-asp-net-co
+
+Folgender Befehl hat mir geholfen um die dotnet ef ausführen zu können. 
+
+```
+export DOTNET_ROLL_FORWARD=LatestMajor
```

Comment:
isn't this related with the currently active dotnet sdk? what sdk version is required, and which ones did it find?